scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController.java

```diff
@@ -61,11 +65,25 @@
 
 		// If there are no validation errors then proceed with persisting the schedules
 		if (!validationErrorResult.hasErrors()) {
+			ApplicationScalingSchedules existingSchedules = scheduleManager.getAllSchedules(app_id);
+			
```

Comment:
And now you show why using the validationErrorResult is so dangerous.
Here you didn't check like you did above.